templates/topbar.php

```diff
@@ -30,9 +18,12 @@ function topbar(): void {
         window.addEventListener('load', () => {
         ReactDOM.render(
             React.createElement(Topbar, {
-                linkData: {
+                links: {
                     notification_api: "<?= $system->router->api_links['notification'] ?>",
                 },
+                notificationAPIData: {
+                    userNotifications: <?= json_encode(NotificationAPIPresenter::userNotificationResponse(notificationManager: $NotificationAPIManager)) ?>,
```

Comment:
```suggestion
                    userNotifications: <?= json_encode(NotificationAPIPresenter::userNotificationResponse($NotificationAPIManager)) ?>,
```
If there's only one parameter we could drop the name for brevity